src/templates.js

```diff
@@ -46,15 +14,30 @@ export function templateProcessor(resource) {
     return [...Array(count).keys()].map(i => {
         const template = Object.assign({}, resource.childrenTemplate);
         template.id = `${template.id}${i}`;
-
-        if (template.output) {
-            template.output = processOutput(i, template.output);
-        }
-
-        if (template.children) {
-            template.children = processChildren(`${template.type}${i}`, template.children);
-        }
-
         return template;
     });
-}
\ No newline at end of file
+}
+
+/*eslint-disable no-unused-vars */
```

Comment:
Let's always put `eslint-disable` statements on a per-line basis instead of over whole files.